Makefile

```diff
@@ -2,16 +2,16 @@
 .PHONY: build clean sdist test
 
 build:
-	pip3 install -r requirements.txt
+	pipenv install
 
 clean:
 	rm -rf sdc/crypto/doc/html
 	rm -v dist/sdc-cryptography-*.tar.gz
 
 sdist:
-	python setup.py sdist
+	pipenv run python setup.py sdist
 
-test: build
-	pip3 install -r test_requirements.txt
-	flake8 .
-	python -m unittest discover sdc
+test:
+	pipenv install --dev
```

Comment:
Again, wouldn't expect this in a makefile.

Reply:
Done